wrangler-service/src/main/java/co/cask/wrangler/service/s3/S3Service.java

```diff
@@ -106,7 +105,7 @@ public void initialize(HttpServiceContext context) throws Exception {
    * @param responder HTTP Response handler.
    */
   @POST
-  @Path("test")
+  @Path("/connections/s3/test")
   public void testS3Connection(HttpServiceRequest request, HttpServiceResponder responder,
                                @PathParam("connection-id") String connectionId) {
```

Comment:
shouldn't this be removed?

Reply:
the connection params should be read from the body of the request right?

Reply:
Since we are removing the `connection-id`, the connection properties should be obtained from the request body, just like adding a connection

Reply:
yeah, my bad. let me fix